tools/picard/picard_macros.xml

```diff
@@ -81,5 +81,7 @@ This will be added shortly
 
     </token>
     
-
+    <token name="@TMP_DIR_FIX@"><!-- Bugfix that ensures that Picards temp dir is set to the galaxy temp dir -->
+        TMP_DIR=$__new_file_path__
```

Comment:
I will try to change and test it asap, although I forsee two problems:
 - When a user/admin installs the picard package and figures out the system is affected by this bug, and TMP_DIR is not set appropriately, the galaxy server must be rebooted to set the environment variable correctly and to make picard work.

- If `\$TMP_DIR` is empty, this argument should not be passed through at all. Would `$__new_file_path__` then be a good alternative?